workers/price-reporter/src/worker.rs

```diff
@@ -59,13 +61,20 @@ impl PriceReporterConfig {
     /// and secret is not provided
     pub(crate) fn exchange_configured(&self, exchange: Exchange) -> bool {
         let disabled = self.disabled_exchanges.contains(&exchange);
-        let configured = match exchange {
-            Exchange::Coinbase => {
-                self.exchange_conn_config.coinbase_api_key.is_some()
-                    && self.exchange_conn_config.coinbase_api_secret.is_some()
-            },
-            Exchange::UniswapV3 => self.exchange_conn_config.eth_websocket_addr.is_some(),
-            _ => true,
+
+        let configured = if self.price_reporter_url.is_some() {
```

Comment:
maybe add a `uses_external_reporter` to the config? Since we use this pattern twice: here and once below. Pretty minor so at your discretion

Reply:
ah, I thought about it, but I'd like to capture one intent with one config option